apps/meteor/server/features/EmailInbox/EmailInbox.ts

```diff
@@ -49,8 +49,8 @@ export async function configureEmailInboxes(): Promise<void> {
 			},
 			{
 				deleteAfterRead: false,
-				filter: [['UNSEEN'], ['SINCE', emailInboxRecord._updatedAt]],
-				rejectBeforeTS: emailInboxRecord._updatedAt,
+				filter: [['UNSEEN'], ['SINCE', emailInboxRecord._createdAt]],
```

Comment:
Any reason to change this? 👀 The idea is just pull messages since the last time the inbox was updated (this was decided long time ago). So if you update the record, you don't try to pull more messages than what's needed.

Reply:
it improves availability, in cases when the admin broke some configuration, waited some time (and e-mails arrived), fixed it, and is expecting the unseen e-mails, this guarantees that it'll deal with these cases, as well as offline for a while -> changes config cases, and it does not increase the load on our server, since the received e-mails are marked as read.